composer.json

```diff
@@ -22,7 +22,7 @@
         "php": ">=7.1.3",
         "consolidation/output-formatters": "^4.1.1",
         "psr/log": "^1|^2",
-        "symfony/console": "^4.4.8|~5.1.0",
+        "symfony/console": "^4.4.8|^5",
```

Comment:
I have not fully reviewed, but if you make this `|| ^5.2`, then it is not neccesary to make the tests work on 5.1 and earlier. We can just require that you take the new symfony/console when you take the new annotated-command. In the unlikely event that a client is constrained to an earlier version of symfony/console, then they will also necessarily be constrained in their annotated-command version. Composer will do this automatically.

Reply:
I force pushed the branch using 5.2 (maybe no good idea, since it is doubled now? will be squashed anyway...).
Any idea about the `The "french" option does not exist.` error?

Reply:
The "french" option is added by an option hook. If that test is not working, then there is something wrong with the hook code, or something wrong with the code that adds options -- probably the later, since other tests would probably fail if the hook code was defective.

I haven't had time to try to diagnose, though. Compare execution of the failing test with and without this PR.